app/serializers/review_serializer.rb

```diff
@@ -1,3 +1,5 @@
 class ReviewSerializer < ActiveModel::Serializer
   attributes :id, :title, :rating, :heatIndex, :body, :created_at
+
+  belongs_to :sauce
```

Comment:
no. We dont need this in your Review serializer. This is going to lead to a circular dependency. delete the line. You can add `sauce_id`, but only if its needed!

Reply:
Removed belongs_to